numpy/_core/src/multiarray/npy_static_data.c

```diff
@@ -184,6 +184,22 @@ initialize_static_globals(void)
         return -1;
     }
 
+    npy_static_pydata.call_kwnames =
```

Comment:
should maybe call all of these `(dlpack|dl)_call_kwnames`, but doens't matter too much.